src/tests/ftest/util/dmg_utils.py

```diff
@@ -73,8 +78,12 @@ def set_hostlist(self, manager):
         Args:
             manager (SubprocessManager): daos server/agent process manager
         """
-        self.hostlist.update(
-            manager.get_config_value("access_points"), "dmg.hostlist")
+        if self.yaml is not None:
```

Comment:
It might be go to consider changing this to: `if self.yaml:` 
Since you'd want to also make sure that self.yaml is set to something "valid". Comparing to None, will allow self.yaml to be set to anything that isn't None like empty list or an empty string.

Reply:
Good call. My python is a bit rusty these days.